tests/integration/golang/aim/run/get_run_metrics_test.go

```diff
@@ -27,13 +30,102 @@ func TestGetRunMetricsTestSuite(t *testing.T) {
 
 func (s *GetRunMetricsTestSuite) SetupTest() {
 	s.BaseTestSuite.SetupTest()
-
-	var err error
-	s.run, err = s.RunFixtures.CreateExampleRun(context.Background(), s.DefaultExperiment)
-	s.Require().Nil(err)
 }
 
 func (s *GetRunMetricsTestSuite) Test_Ok() {
+	// create test data
+	experiment, err := s.ExperimentFixtures.CreateExperiment(context.Background(), &models.Experiment{
+		Name:           uuid.New().String(),
+		NamespaceID:    s.DefaultNamespace.ID,
+		LifecycleStage: models.LifecycleStageActive,
+	})
+	s.Require().Nil(err)
+
+	run, err := s.RunFixtures.CreateRun(context.Background(), &models.Run{
+		ID:             "id",
+		Name:           "TestRun",
+		Status:         models.StatusScheduled,
+		StartTime:      sql.NullInt64{Int64: 123456789, Valid: true},
+		EndTime:        sql.NullInt64{Int64: 123456789, Valid: true},
+		SourceType:     "JOB",
+		ExperimentID:   *experiment.ID,
+		LifecycleStage: models.LifecycleStageActive,
+	})
+	s.Require().Nil(err)
+
+	// create context and attach it to own metric.
+	metricContext, err := s.ContextFixtures.CreateContext(context.Background(), &models.Context{
+		Json: datatypes.JSON(`{"key1": "key1", "value1": "value1"}`),
+	})
+	s.Require().Nil(err)
+
+	_, err = s.MetricFixtures.CreateMetric(context.Background(), &models.Metric{
+		Key:       "key1",
+		Value:     123.1,
+		Timestamp: 123456789,
+		Step:      1,
+		IsNan:     false,
+		RunID:     run.ID,
+		Iter:      1,
+		ContextID: common.GetPointer(metricContext.ID),
```

Comment:
see comment above -- might be able to simplify

Reply:
yep.